assets/chef-recipes/cookbooks/docker/libraries/docker_installation_package.rb

```diff
@@ -130,12 +138,30 @@ def focal?
       false
     end
 
+    def jammy?
+      node['platform'] == 'ubuntu' && node['platform_version'] == '22.04'
+    end
 
     def amazon?
       return true if node['platform'] == 'amazon'
       false
     end
 
+    def arm?
+      node[:kernel][:machine] == 'aarch64'
+    end
+
+    def arch_alias(architecture)
+      case architecture
+      when 'aarch64'
+        return 'arm64'
+      when 'x86_64'
+        return 'amd64'
+      else
+        return architecture
+      end
+    end
+
     # https://github.com/chef/chef/issues/4103
```

Comment:
See the original issue. It should be resolved in the Chef version that we are using.